src/psyclone/tests/domain/lfric/lfric_field_mdata_test.py

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2017-2022, Science and Technology Facilities Council.
+# Copyright (c) 2017-2023, Science and Technology Facilities Council.
```

Comment:
If we agree to drop logical fields then I think the changes to this file can be reverted.